test/Dialect/Calyx/backedges.mlir

```diff
@@ -0,0 +1,16 @@
+// RUN: circt-opt --split-input-file %s | FileCheck %s
+
```

Comment:
Can you add a comment explaining what this test is looking for?

Reply:
Sure! this is mainly just to sanity check the graph region, but thinking about it, i think it might fit better into the rount-trip test - doesn't really warrant a stand-alone test.